__tests__/keys.test.ts

```diff
@@ -65,16 +65,19 @@ describe('Key generation functions', () => {
       // Make sure that the mapping from buffer to key pair doesn't change unexpectedly.
       expect(keyPairFromData(Buffer.from('0'))).toEqual({
         publicKey: '069a33d37101d7089b606f92e4b41553c237a474ad9d6f62eeb6708415f98f4d',
+        publicKeyYCoordinate: '0717e78b98a53888aa7685b91137fa01b9336ce7d25f874dbfb8d752c6ac610d',
```

Comment:
These tests weren't running in CI before. Simply added what it was returning for `publicKeyYCoordinate`